dev/archery/archery/cli.py

```diff
@@ -586,14 +585,29 @@ def _get_comparisons_as_json(comparisons):
     return buf.getvalue()
 
 
-def _format_comparisons_with_pandas(comparisons_json):
+def _format_comparisons_with_pandas(comparisons_json, no_counters):
     import pandas as pd
```

Comment:
@kszucs I don't see `pandas` in dev/archery/requirements.txt or ci/conda_env_archery.yml. Should it be added?

Reply:
+1